server/src/main/java/io/unitycatalog/server/service/PrivilegeAssignment.java

```diff
@@ -0,0 +1,13 @@
+package io.unitycatalog.server.service;
+
+import io.unitycatalog.server.model.Privilege;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+class PrivilegeAssignment {
```

Comment:
should we define this in the spec and let it be autogenerated?

Reply:
Ya. Not a bad idea.